docs/docusaurus/docs/cloud/deploy_gx_agent.md

```diff
@@ -0,0 +1,244 @@
+---
+sidebar_label: 'Deploy the GX Agent'
+title: 'Deploy the GX Agent'
+id: deploy_gx_agent
+description: Deploy the GX Agent to use GX Cloud features and functionality.
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+To use GX Cloud features and functionality, you need to deploy the GX Agent. The GX Agent runs open source GX code in GX Cloud, and it allows you to securely access your data without connecting to it or interacting with it directly. To learn more about the GX Agent and deployment patterns, see [About GX Cloud](./about_gx.md).
```

Comment:
Should this be more similar to how we describe the GX Agent under About? 
> The GX Agent is an intermediary between GX Cloud and your organization's data stores. GX Cloud does not connect directly to your data, all data access occurs within the GX Agent. GX Cloud sends jobs to the GX Agent, the GX Agent executes these jobs against your data, and then sends the job results to GX Cloud.

>The GX Agent is typically deployed in your organization's deployment environment, for example, in a development, staging, or production cloud services environment. The GX Agent serves all GX Cloud users within your organization. It can be run as part of your development or production workflows.

Reply:
This phrase in particular feels not quite right, since we run more than just OSS code and the code is not run _in_ GX Cloud necessarily, since it has to reach back to our internal database for some operations:
> The GX Agent runs open source GX code in GX Cloud

Reply:
@annabarr I've updated the intro text as recommended and the statement about running open source code no longer appears in the text.